{
  "flakyProjects" : [ {
    "projectName" : "Quarkus QE TS: Quarkus Picocli",
    "projectBaseDir" : "quarkus-picocli",
    "flakeTests" : [ {
      "fullTestName" : "PicocliDevIT.verifyGreetingCommandOutputsExpectedMessage",
      "flakes" : [ {
        "failureMessage" : "Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.",
        "failureType" : "org.awaitility.core.ConditionTimeoutException",
        "failureStackTrace" : "org.awaitility.core.ConditionTimeoutException: Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.\n\tat org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)\n\tat org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1006)\n\tat org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:790)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:134)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:124)\n\tat io.quarkus.test.utils.LogsVerifier.assertContains(LogsVerifier.java:28)\n\tat PicocliDevIT.verifyGreetingCommandOutputsExpectedMessage(PicocliDevIT.java:61)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\nCaused by: java.lang.NullPointerException: Cannot invoke \"io.quarkus.test.logging.LoggingHandler.logs()\" because \"this.loggingHandler\" is null\n\tat io.quarkus.test.services.quarkus.LocalhostQuarkusApplicationManagedResource.logs(LocalhostQuarkusApplicationManagedResource.java:112)\n\tat io.quarkus.test.bootstrap.BaseService.getLogs(BaseService.java:194)\n\tat io.quarkus.test.utils.LogsVerifier.lambda$assertContains$0(LogsVerifier.java:29)\n\tat org.awaitility.core.AssertionCondition.lambda$new$0(AssertionCondition.java:53)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:248)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:235)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:840)\n",
        "dateTime" : "2024-09-25T01:40:30.320946627Z[UTC]",
        "ciJobName" : "quarkus-main-rhel8-jdk17-baremetal-ts-jvm-podman-nightly",
        "ciBuildNumber" : "364"
      }, {
        "failureMessage" : "Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.",
        "failureType" : "org.awaitility.core.ConditionTimeoutException",
        "failureStackTrace" : "org.awaitility.core.ConditionTimeoutException: Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.\n\tat org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)\n\tat org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1006)\n\tat org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:790)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:134)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:124)\n\tat io.quarkus.test.utils.LogsVerifier.assertContains(LogsVerifier.java:28)\n\tat PicocliDevIT.verifyGreetingCommandOutputsExpectedMessage(PicocliDevIT.java:61)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\nCaused by: java.lang.NullPointerException: Cannot invoke \"io.quarkus.test.logging.LoggingHandler.logs()\" because \"this.loggingHandler\" is null\n\tat io.quarkus.test.services.quarkus.LocalhostQuarkusApplicationManagedResource.logs(LocalhostQuarkusApplicationManagedResource.java:112)\n\tat io.quarkus.test.bootstrap.BaseService.getLogs(BaseService.java:194)\n\tat io.quarkus.test.utils.LogsVerifier.lambda$assertContains$0(LogsVerifier.java:29)\n\tat org.awaitility.core.AssertionCondition.lambda$new$0(AssertionCondition.java:53)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:248)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:235)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:840)\n",
        "dateTime" : "2024-09-25T01:40:30.320946627Z[UTC]",
        "ciJobName" : "quarkus-main-rhel8-jdk17-baremetal-ts-jvm-podman-nightly",
        "ciBuildNumber" : "364"
      }, {
        "failureMessage" : "Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.",
        "failureType" : "org.awaitility.core.ConditionTimeoutException",
        "failureStackTrace" : "org.awaitility.core.ConditionTimeoutException: Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.\n\tat org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)\n\tat org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1006)\n\tat org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:790)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:134)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:124)\n\tat io.quarkus.test.utils.LogsVerifier.assertContains(LogsVerifier.java:28)\n\tat PicocliDevIT.verifyGreetingCommandOutputsExpectedMessage(PicocliDevIT.java:61)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\nCaused by: java.lang.NullPointerException: Cannot invoke \"io.quarkus.test.logging.LoggingHandler.logs()\" because \"this.loggingHandler\" is null\n\tat io.quarkus.test.services.quarkus.LocalhostQuarkusApplicationManagedResource.logs(LocalhostQuarkusApplicationManagedResource.java:112)\n\tat io.quarkus.test.bootstrap.BaseService.getLogs(BaseService.java:194)\n\tat io.quarkus.test.utils.LogsVerifier.lambda$assertContains$0(LogsVerifier.java:29)\n\tat org.awaitility.core.AssertionCondition.lambda$new$0(AssertionCondition.java:53)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:248)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:235)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:840)\n",
        "dateTime" : "2024-09-25T01:40:30.320946627Z[UTC]",
        "ciJobName" : "quarkus-main-rhel8-jdk17-baremetal-ts-jvm-podman-nightly",
        "ciBuildNumber" : "364"
      }, {
        "failureMessage" : "Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.",
        "failureType" : "org.awaitility.core.ConditionTimeoutException",
        "failureStackTrace" : "org.awaitility.core.ConditionTimeoutException: Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.\n\tat org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)\n\tat org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1006)\n\tat org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:790)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:134)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:124)\n\tat io.quarkus.test.utils.LogsVerifier.assertContains(LogsVerifier.java:28)\n\tat PicocliDevIT.verifyGreetingCommandOutputsExpectedMessage(PicocliDevIT.java:61)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\nCaused by: java.lang.NullPointerException: Cannot invoke \"io.quarkus.test.logging.LoggingHandler.logs()\" because \"this.loggingHandler\" is null\n\tat io.quarkus.test.services.quarkus.LocalhostQuarkusApplicationManagedResource.logs(LocalhostQuarkusApplicationManagedResource.java:112)\n\tat io.quarkus.test.bootstrap.BaseService.getLogs(BaseService.java:194)\n\tat io.quarkus.test.utils.LogsVerifier.lambda$assertContains$0(LogsVerifier.java:29)\n\tat org.awaitility.core.AssertionCondition.lambda$new$0(AssertionCondition.java:53)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:248)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:235)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:840)\n",
        "dateTime" : "2024-09-25T01:40:30.320946627Z[UTC]",
        "ciJobName" : "quarkus-main-rhel8-jdk17-baremetal-ts-jvm-podman-nightly",
        "ciBuildNumber" : "364"
      } ]
    }, {
      "fullTestName" : "PicocliDevIT.verifyErrorForMultipleCommandsWithoutTopCommand",
      "flakes" : [ {
        "failureMessage" : "Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.",
        "failureType" : "org.awaitility.core.ConditionTimeoutException",
        "failureStackTrace" : "org.awaitility.core.ConditionTimeoutException: Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.\n\tat org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)\n\tat org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1006)\n\tat org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:790)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:134)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:124)\n\tat io.quarkus.test.utils.LogsVerifier.assertContains(LogsVerifier.java:28)\n\tat PicocliDevIT.verifyErrorForMultipleCommandsWithoutTopCommand(PicocliDevIT.java:96)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\nCaused by: java.lang.NullPointerException: Cannot invoke \"io.quarkus.test.logging.LoggingHandler.logs()\" because \"this.loggingHandler\" is null\n\tat io.quarkus.test.services.quarkus.LocalhostQuarkusApplicationManagedResource.logs(LocalhostQuarkusApplicationManagedResource.java:112)\n\tat io.quarkus.test.bootstrap.BaseService.getLogs(BaseService.java:194)\n\tat io.quarkus.test.utils.LogsVerifier.lambda$assertContains$0(LogsVerifier.java:29)\n\tat org.awaitility.core.AssertionCondition.lambda$new$0(AssertionCondition.java:53)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:248)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:235)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:840)\n",
        "dateTime" : "2024-09-25T01:40:30.320521474Z[UTC]",
        "ciJobName" : "quarkus-main-rhel8-jdk17-baremetal-ts-jvm-podman-nightly",
        "ciBuildNumber" : "364"
      }, {
        "failureMessage" : "Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.",
        "failureType" : "org.awaitility.core.ConditionTimeoutException",
        "failureStackTrace" : "org.awaitility.core.ConditionTimeoutException: Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.\n\tat org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)\n\tat org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1006)\n\tat org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:790)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:134)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:124)\n\tat io.quarkus.test.utils.LogsVerifier.assertContains(LogsVerifier.java:28)\n\tat PicocliDevIT.verifyErrorForMultipleCommandsWithoutTopCommand(PicocliDevIT.java:96)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\nCaused by: java.lang.NullPointerException: Cannot invoke \"io.quarkus.test.logging.LoggingHandler.logs()\" because \"this.loggingHandler\" is null\n\tat io.quarkus.test.services.quarkus.LocalhostQuarkusApplicationManagedResource.logs(LocalhostQuarkusApplicationManagedResource.java:112)\n\tat io.quarkus.test.bootstrap.BaseService.getLogs(BaseService.java:194)\n\tat io.quarkus.test.utils.LogsVerifier.lambda$assertContains$0(LogsVerifier.java:29)\n\tat org.awaitility.core.AssertionCondition.lambda$new$0(AssertionCondition.java:53)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:248)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:235)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:840)\n",
        "dateTime" : "2024-09-25T01:40:30.320521474Z[UTC]",
        "ciJobName" : "quarkus-main-rhel8-jdk17-baremetal-ts-jvm-podman-nightly",
        "ciBuildNumber" : "364"
      }, {
        "failureMessage" : "Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.",
        "failureType" : "org.awaitility.core.ConditionTimeoutException",
        "failureStackTrace" : "org.awaitility.core.ConditionTimeoutException: Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.\n\tat org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)\n\tat org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1006)\n\tat org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:790)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:134)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:124)\n\tat io.quarkus.test.utils.LogsVerifier.assertContains(LogsVerifier.java:28)\n\tat PicocliDevIT.verifyErrorForMultipleCommandsWithoutTopCommand(PicocliDevIT.java:96)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\nCaused by: java.lang.NullPointerException: Cannot invoke \"io.quarkus.test.logging.LoggingHandler.logs()\" because \"this.loggingHandler\" is null\n\tat io.quarkus.test.services.quarkus.LocalhostQuarkusApplicationManagedResource.logs(LocalhostQuarkusApplicationManagedResource.java:112)\n\tat io.quarkus.test.bootstrap.BaseService.getLogs(BaseService.java:194)\n\tat io.quarkus.test.utils.LogsVerifier.lambda$assertContains$0(LogsVerifier.java:29)\n\tat org.awaitility.core.AssertionCondition.lambda$new$0(AssertionCondition.java:53)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:248)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:235)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:840)\n",
        "dateTime" : "2024-09-25T01:40:30.320521474Z[UTC]",
        "ciJobName" : "quarkus-main-rhel8-jdk17-baremetal-ts-jvm-podman-nightly",
        "ciBuildNumber" : "364"
      }, {
        "failureMessage" : "Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.",
        "failureType" : "org.awaitility.core.ConditionTimeoutException",
        "failureStackTrace" : "org.awaitility.core.ConditionTimeoutException: Assertion condition defined as a Lambda expression in io.quarkus.test.utils.LogsVerifier null within 30 seconds.\n\tat org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)\n\tat org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)\n\tat org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1006)\n\tat org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:790)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:134)\n\tat io.quarkus.test.utils.AwaitilityUtils.untilAsserted(AwaitilityUtils.java:124)\n\tat io.quarkus.test.utils.LogsVerifier.assertContains(LogsVerifier.java:28)\n\tat PicocliDevIT.verifyErrorForMultipleCommandsWithoutTopCommand(PicocliDevIT.java:96)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\nCaused by: java.lang.NullPointerException: Cannot invoke \"io.quarkus.test.logging.LoggingHandler.logs()\" because \"this.loggingHandler\" is null\n\tat io.quarkus.test.services.quarkus.LocalhostQuarkusApplicationManagedResource.logs(LocalhostQuarkusApplicationManagedResource.java:112)\n\tat io.quarkus.test.bootstrap.BaseService.getLogs(BaseService.java:194)\n\tat io.quarkus.test.utils.LogsVerifier.lambda$assertContains$0(LogsVerifier.java:29)\n\tat org.awaitility.core.AssertionCondition.lambda$new$0(AssertionCondition.java:53)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:248)\n\tat org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:235)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:840)\n",
        "dateTime" : "2024-09-25T01:40:30.320521474Z[UTC]",
        "ciJobName" : "quarkus-main-rhel8-jdk17-baremetal-ts-jvm-podman-nightly",
        "ciBuildNumber" : "364"
      } ]
    } ]
  }, {
    "projectName" : "Quarkus QE TS: Test Tooling: Pact",
    "projectBaseDir" : "test-tooling/pact",
    "flakeTests" : [ {
      "fullTestName" : "io.quarkus.ts.http.pact.FlakyTest.failOnce",
      "flakes" : [ ]
    } ]
  }, {
    "projectName" : "Quarkus QE TS: Build Time Analytics",
    "projectBaseDir" : "build-time-analytics",
    "flakeTests" : [ {
      "fullTestName" : "io.quarkus.ts.buildtimeanalytics.CliDevModeIT.timeoutPropertyRecognized",
      "flakes" : [ ]
    } ]
  } ]
}